hack/arktos-up-scale-out-poc.sh

```diff
@@ -530,7 +547,9 @@ if [[ -n "${PSP_ADMISSION}" && "${AUTHORIZATION_MODE}" = *RBAC* ]]; then
 fi
 
 if [[ "${DEFAULT_STORAGE_CLASS}" = "true" ]]; then
```

Comment:
= -> ==?

Reply:
oh? it looks a bug from the community local-up-cluster.sh!

Anyway, I just updated hack/arktos-up-scale-out-poc.sh for now.